pulser-simulation/pulser_simulation/qutip_result.py

```diff
@@ -188,8 +205,9 @@ def get_state(
                 )
             ex_inds = [
                 i
-                for i in range(3**self._size)
-                if ex_state in np.base_repr(i, base=3).zfill(self._size)
+                for i in range(self._dim**self._size)
+                if ex_state
+                in np.base_repr(i, base=self._dim).zfill(self._size)
```

Comment:
I am only allowing to reduce to either "ground-rydberg", "digital" or "XY". I don't know if it would make sense to reduce to "all" or to "ground-rydberg_with_error" or "digital_with_error" ?

Reply:
We can leave that for later, for now those three are fine